strategy/live_trader.py

```
@@ -0,0 +1,714 @@
+            try:
+                raw = m.get("clobTokenIds")
+                import json
+                tids = json.loads(raw) if isinstance(raw, str) else raw
+                if not tids or len(tids) != 2:
+                    continue
+                st = _iso_to_unix(m.get("eventStartTime") or "")
+                et = _iso_to_unix(m.get("endDate") or m.get("endDateIso") or "")
+                if st <= now < et:
+                    candidates.append((st, et, m, tids))
+            except Exception:
+                continue
```

Ruff (0.16.3)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)